src/OpenTelemetry.AutoInstrumentation/Instrumentation.cs

```diff
@@ -242,6 +244,11 @@ private static void AddLazilyLoadedMetricInstrumentations(LazyInstrumentationLoa
                     break;
                 default:
                     Logger.Warning($"Configured metric instrumentation type is not supported: {instrumentation}");
+                    if (FailFastSettings.Value.FailFast)
```

Comment:
Will this be overridden if the FAIL FAST environment variable is set? Is there already a smoke test that covers this scenario?